Content.Server/Thief/Systems/ThiefBeaconSystem.cs

```diff
@@ -37,7 +39,9 @@ private void OnGetInteractionVerbs(Entity<ThiefBeaconComponent> beacon, ref GetV
             return;
 
         var mind = _mind.GetMind(args.User);
-        if (!HasComp<ThiefRoleComponent>(mind))
+        if (mind == null)
+            return;
+        if (!_roles.MindHasRole<ThiefRoleComponent>(mind!.Value))
             return;
```

Comment:
You can combine the two if statements with an OR operator

if (mind == null || !_roles.MindHasRole<ThiefRoleComponent>(mind.Value))
            return;